### ISSUE TITLE
iron-fit-behavior accessibility issues 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-fit-behavior#10]
Audit results: 
[Warning] Text elements should have a reasonable contrast ratio (1)
Not applicable tests (14)

Contrast: can we make the shade of the text just a bit darker to make the contrast better against the background? 

It appears that none of this should be interactive, so keyboard navigability is NA here.
